knowledge_storm/lm.py

```diff
@@ -37,7 +37,7 @@
     litellm.drop_params = True
     litellm.telemetry = False
 
-from litellm.caching.caching import Cache
+from litellm.caching import Cache
```

Comment:
which litellm version are you using? Above version works for `1.49.3`

Is this change necessary to make the code work for latest version?

Reply:
Ok, I was using '1.48.0'. I can rollback this change but I am actually verifying `requirements.tx`.

Reply:
Reverted the change.